public/demos/oversized-images.html

```diff
@@ -19,9 +19,9 @@ <h2 id="feature-allowed-banner">
   <span id="allowfeature">...</span>
 </h2>
 
-<p>Click the disallow/allow buttons above to see how the page loads with the policy on and off.
-  When the policy is on, you will see images that have intrinsic dimentions which are much larger
-  than their container size render as placeholder images.</p>
+<p>Click the buttons above to see how the page loads with different policy values.
+  Images that have image_size / container_size ratio exceed given threshold should
```

Comment:
Nit: When I see |image_size| and |container_size| they sound like properties that should exist on an element or API somewhere. I don't think that's the case, so I'd suggest using a less technical explanation.

```suggestion
  Images where the ratio of intrinsic dimensions to container size exceeds the given threshold should
```

Reply:
Done.